CMakeLists.txt

```diff
@@ -2,8 +2,16 @@ cmake_minimum_required(VERSION 3.15)
 
 include(CheckSymbolExists)
 include(CheckIPOSupported)
+include(CMakeDependentOption)
 
-project(ninja)
+project(ninja
+	LANGUAGES C CXX
+	DESCRIPTION "Ninja is a small build system with a focus on speed."
+	HOMEPAGE_URL "https://ninja-build.org/"
+	VERSION 1.10.2
+)
+
+cmake_dependent_option(create_msi "Create .msi installer" OFF WIN32 OFF)
```

Comment:
An alternative to making a new option would be to always run the cpack code.

Generating the cpack output doesn't create the package, something still has to call cpack.

You would still have to gate the cpack code this PR adds on the WIX generator being available though.